substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -217,10 +217,6 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxExposurePageSize: Get<u32>;
 
-		/// The fraction of the validator set that is safe to be offending.
-		/// After the threshold is reached a new era will be forced.
-		type OffendingValidatorsThreshold: Get<Perbill>;
```

Comment:
Instead of removing this storage item, why not setting it to 0%? Again, it seems that we're losing generality and flexibility in this PR without a convincing reason for that.

Reply:
Forcing a new era is not only not necessary, it can be detrimental. In parachain consensus we have to assume (unfortunately) that determinism can be attacked, hence it can happen that a malicious actor gets honest nodes slashed and disabled. We designed disabling in a way that it will never be a security threat, even if honest nodes get disabled due to an attack.

We are also careful with our slashes to limit the likelihood of honest nodes getting kicked out in a next election or at least, if it happens that the attacker will lose at least as many nodes himself. Regardless, the safest is disabling, a new era imposes a higher risk, hence we would not like to rush it. The more time governance has to react, the better.

Reply:
Sounds good, thanks for the clarifications.